### DISCUSSION TITLE
Installare HA su raspberry pi400

### DISCUSSION THREAD

**Discussion by gigigentile** [indomus/forum#1120] [HUB: Home Assistant]
Un saluto a tutto il forum!
Sono in procinto di addentrarmi nel mondo della automazione domestica da neofita, anche se ho alle spalle una vita
passata con il saldatore in mano e, dai tempi del Commodore 64, a programmare in assembler e in vari linguaggi a basso livello!
Mi chiedevo se è possibile installare appunto la versione HA Core su un raspberry pi400, dato che è l'ultimo arrivato della famiglia
raspy e gli altri 3 sono impegnati in altre mansioni (Home Center, DAW etc...).
Qualcuno l'ha già fatto?
Grazie per l'attenzione!

**Comment by KC-inDomus** [comment]
sì che puoi, alla fine è un Rapsberry pure lui. 
segui passo passo [l'iter standard](https://indomus.it/formazione/home-assistant-su-raspberry-pi-varie-installazioni-le-cose-da-fare/) e vai dritto senza problemi.

**Reply by gigigentile** [reply]
Grazie!
Mi accingo a provare... e vi tengo informati!
